guides/documentation/service-interceptors.md

```diff
@@ -149,6 +149,39 @@ functions of further interceptors in the path are invoked as if no
 pause had occurred. A single context may pause and resume an arbitrary
 number of times.
 
+### Examples
+
+Here's how you define a "before" interceptor:
+
+```clojure
+(ns interceptor.example
+  (:require [io.pedestal.interceptor :refer [interceptor]]))
+
+(def my-before-interceptor
+  (interceptor
+   {:name ::hello-world
+    :enter
+    (fn [context]
+      (assoc context :response 
+                     {:status 200 :body "Hello world!" 
+                      :headers {"Content-Type" "text/plain"}}))}))
```

Comment:
By "Interceptor API" does than mean `io.pedestal.http/interceptor` alone, or does that include the helper functions in `io.pedestal.interceptor.helpers`? We make pretty extensive use of the `before` function in our codebase, so I'd be sad to see it go.